docker/dev/supervisor.conf

```diff
@@ -0,0 +1,11 @@
+programs:
+  - name: "vpp"
+    executable-path: "/usr/bin/vpp"
+    executable-args: ["-c", "/etc/vpp/vpp.conf"]
+    logfile-path: "/var/log/supervisord.log"
+  - name: "agent"
+    executable-path: "/go/bin/vpp-agent"
+    executable-args: ["--config-dir=/opt/vpp-agent/dev"]
+    logfile-path: "/var/log/supervisord.log"
```

Comment:
Neither is this one. I don't think logfile paths should be set at all. We want to have log output coming from vpp-agent-init's stdout/stderr, just like it was before.

Reply:
Previously we had both, a log file and stdout/stderr. Let's stay only with the latter.

Reply:
We actually did not:

https://github.com/ligato/vpp-agent/blob/80401e66a0ef370ac61c404590de51d86f78bf4c/docker/dev/supervisord.conf#L7-L18

Reply:
I see, the log file set was only for the supervisory process

Reply:
The confusion might be because the `/var/log/supervisord.log` file contained output coming from redirected stdout/stderr.